queries/bash/highlights.scm

```diff
@@ -144,6 +144,12 @@
              (concatenation (word) @parameter)
              ])
 
+(command
+  name: (command_name) @_name
+  .
+  argument: (word) @function.call
+  (#eq? @_name "command"))
```

Comment:
I assume this doesn't cover `command -[pVv] foo`.
There are also other similar commands like `which`, `whereis`, `hash`, `sudo`, `pkexec` …
So while I'm not against this change, it might be too complicated to properly implement.

Reply:
Dang, you're right. I am fine to close this then :+1: